WordPressUI/BottomSheet/BottomSheetViewController.swift

```diff
@@ -10,8 +10,8 @@ public class BottomSheetViewController: UIViewController {
         /// The height of the space above the bottom sheet content, including the grip view and space around it.
         ///
         public static let additionalContentTopMargin: CGFloat = BottomSheetViewController.Constants.gripHeight
-            + BottomSheetViewController.Constants.Header.spacing
-            + BottomSheetViewController.Constants.Stack.insets.top
+        + BottomSheetViewController.Constants.Header.spacing
+                    + BottomSheetViewController.Constants.Stack.insets.top
```

Comment:
Was this change intentional? Seems like things would look misaligned now?

Reply:
Hm, definitely not intentional. Fixed: 7696cdbf434b5da54f835c2a85d25a2e2774352d